content-src/components/PocketStories/PocketStories.js

```diff
@@ -59,7 +59,7 @@ const PocketStories = React.createClass({
 
     return (<section className="pocket-stories spotlight">
       <h3 className="section-title">
-        <FormattedMessage id="header_stories" />
+        Trending Stories By
```

Comment:
@Mardak we just had some wires crossed here. I will change this commit to add a localization note to header_stories..